robyn/dev_event_handler.py

```diff
@@ -7,11 +8,14 @@ class EventHandler(FileSystemEventHandler):
     def __init__(self, file_name) -> None:
         self.file_name = file_name
         self.processes = []
+        self.alias = "python3" if not sys.platform.startswith("win32") else "python"
```

Comment:
One last nit: can you name it as `python_alias`?

Reply:
Done